uber/config.py

```diff
@@ -131,6 +143,16 @@ def __getattr__(self, name):
                 return sa.localized_now() > date_setting
         elif name.startswith('HAS_') and name.endswith('_ACCESS'):
             return getattr(c, name.split('_')[1]) in sa.AdminAccount.access_set()
+        elif name.endswith('_AVAILABLE'):
+            item_check = name.split('_', 1)[0]
```

Comment:
Good to know, thanks.  As you can see, I've been using ``self`` because of the same logic, but it's nice to have a sanity check.